pyairtable/api/api.py

```diff
@@ -17,17 +18,24 @@ class Api(ApiAbstract):
         >>> api.all('base_id', 'table_name')
     """
 
-    def __init__(self, api_key: str, timeout=None):
+    def __init__(
+        self,
+        api_key: str,
+        timeout=None,
+        max_records_per_request=DEFAULT_MAX_RECORDS_PER_REQUEST,
+    ):
         """
 
         Args:
             api_key: |arg_api_key|
 
         Keyword Args:
             timeout(``Tuple``): |arg_timeout|
-
+            max_records_per_request: |arg_max_records_per_request|
         """
-        super().__init__(api_key, timeout=timeout)
+        super().__init__(
+            api_key, max_records_per_request=max_records_per_request, timeout=timeout
+        )
```

Comment:
Carrying through my suggestions for keyword-only arguments...
```suggestion
        super().__init__(api_key, **kwargs)
```